config/rendered-items-assets-template.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Rendered Items Assets Html</title>
+    <style><%= styles %></style>
+  </head>
+  <body>
+    <div class="a">.</div>
+    <div class="b">.</div>
+    <div class="c">.</div>
+    <div class="d">.</div>
+    <div class="e">.</div>
+    <div class="f">.</div>
+    <div class="g">.</div>
+    <div class="h">.</div>
+    <div class="i">.</div>
+    <div class="j">.</div>
+    <div class="k">.</div>
+    <div class="l">.</div>
+    <div class="m">.</div>
+    <div class="n">.</div>
+    <div class="o">.</div>
+    <div class="p">.</div>
+    <div class="q">.</div>
+    <div class="r">.</div>
+    <div class="s">.</div>
+    <div class="t">.</div>
+    <div class="u">.</div>
```

Comment:
So close... I'm sure we can find some more in fontastique?

Reply:
maybe we use font names for classes to avoid lookups?

Reply:
@webb04 👍 I've renamed the classes. I've taken reference from the `styles.ts` file. There were some duplicate font urls so pruned those.